doc/releases/changelog-dev.md

```diff
@@ -35,6 +35,12 @@
   
 <h3>Breaking changes</h3>
 
+<h3>Bug fixes</h3>
+
+* `QNGOptimizer` did not work with operators whose generator was a Hamiltonian.
+  [(#2524)](https://github.com/PennyLaneAI/pennylane/pull/2524)
```

Comment:
```suggestion
* Fixed a bug where `QNGOptimizer` did not work with operators
  whose generator was a Hamiltonian.
  [(#2524)](https://github.com/PennyLaneAI/pennylane/pull/2524)
```

Very minor, but it might be nice to include 'Fixed' in there as well :)